titus-common/src/main/java/com/netflix/titus/common/runtime/SystemLogService.java

```diff
@@ -0,0 +1,24 @@
+package com.netflix.titus.common.runtime;
+
+/**
+ * An API for logging critical system events (errors, exceptions, invariant violations) with supplementary metadata information.
+ * The events are classified by their category and priority for better searchability.
+ */
+public interface SystemLogService {
```

Comment:
This class needs to set expectations for this will be used including examples for each type and the guidelines for when you choose a different priority or level.